news/_posts/2023-10-10-release-20231010.md

```diff
@@ -0,0 +1,44 @@
+---
+
+With two months of successful playtesting behind us, we are pleased to finally announce [Release 20231010]({{ '/download' | relative_url }}), which brings the public release up to date with development changes from November 2022 through to August 2023. The new release contains some long-awaited UI and UX improvements and many polish changes overall. This release serves as a jump off point for many of the larger upcoming features, so do expect great things in the future!
+
+<figure>
+   <video autoplay loop muted>
+    <source src="{{ '/images/news/20231010-darkcolors.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20231010-darkcolors.webm' | relative_url }}" type="video/webm">
+    <img src="{{ '/images/news/20231010-darkcolors.png' | relative_url }}"/>
+  </video>
+  <figcaption>Restored the ability to select darker player colors.</figcaption>
+</figure>
+
+For more information see the other newsposts [[one]({{ '/news/playtest-20230801/' | relative_url }}), [two]({{ '/news/playtest-20230808/' | relative_url }})], [three]({{ '/news/playtest-20230927/' | relative_url }})] and, as always, for a full overview - [the full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/08829bc0e6a1bf62aa7797fcfc0a9810d27e81d2).
+
+While OpenRA's releases have slowed down due to lack of manpower, our third party scene is going strong! An OpenRA mod & standalone game [Combined Arms](https://www.moddb.com/mods/command-conquer-combined-arms) was the second most popular on ModDB in June this year. It includes units from various C&C games and has 5 factions, each having 5 subfactions. Its newest installment contains 25 singleplayer missions. [DORF Real-Time Strategic Conflict](https://www.dorf-rts.com), a game based on OpenRA's engine has become one of the most anticipated RTS's. We highly encourage everyone to checkout its gameplay trailer as well as [other videos](https://www.youtube.com/@dorf-rts).
+
+<figure>
+  <lite-youtube videoid="1OY_k4S2LVY"></lite-youtube>
+  <img src="{{ '/images/news/20231010-dorf.png' | relative_url }}"/>
+</figure>
+
+Other popular mods include: [Shattered Paradise](https://www.moddb.com/mods/shattered-paradise) an expansion for Tiberian Sun on OpenRA which in addition to GDI and Nod has Mutant, C.A.B.A.L. and Scrin factions. It is a gorgeous and over the top game with a heavy Command and Conquer 3 influence that attempts to add more variety while preserving the vanilla TS feel. A continuation of the Hard Vacuum project [OpenHV](https://openhv.itch.io/openhv). Originally inspired by Dune 2, but with its ambitious goals it never left the development stage and had its assets released under a free license. Now the mantle has been taken up by open source developers. [Romanov's Vengeance](https://www.moddb.com/mods/romanovs-vengeance) is based on OpenRA's work-in-progress Red Alert 2 branch. Its units and balance is heavily inspired by Command & Conquer: Generals.
```

Comment:
```suggestion
Other popular mods include: [Shattered Paradise](https://www.moddb.com/mods/shattered-paradise) - an expansion for Tiberian Sun on OpenRA which in addition to GDI and Nod has Mutant, C.A.B.A.L. and Scrin factions. It is a gorgeous and over the top game with a heavy Command and Conquer 3 influence that attempts to add more variety while preserving the vanilla TS feel. [OpenHV](https://openhv.itch.io/openhv) - a continuation of the Hard Vacuum project, which was originally inspired by Dune 2, but with its ambitious goals it never left the development stage and had its assets released under a free license. Now the mantle has been taken up by open source developers. [Romanov's Vengeance](https://www.moddb.com/mods/romanovs-vengeance) is based on OpenRA's work-in-progress Red Alert 2 mod. Its units and balance is heavily inspired by Command & Conquer: Generals.
```

Reply:
I specifically wanted to break the pattern here